src/olympia/reviewers/management/commands/review_reports_sql/addon/breakdown.sql

```diff
@@ -2,30 +2,46 @@ SELECT 'All Reviewers' AS `Group`,
        IFNULL(FORMAT(SUM(aa.weight), 0), '-') AS `Total Risk`,
        IFNULL(FORMAT(AVG(aa.weight), 2), '-') AS `Average Risk`,
        FORMAT(COUNT(*), 0) AS `Add-ons Reviewed`
-FROM `reviewer_scores` `rs`
-LEFT JOIN `editors_autoapprovalsummary` `aa` ON `aa`.`version_id` = `rs`.`version_id`
-JOIN `users` `u` ON `u`.`id` = `rs`.`user_id`
-WHERE DATE(`rs`.`created`) BETWEEN @WEEK_BEGIN AND @WEEK_END
-  AND `u`.`deleted` = 0
+FROM (
+  SELECT DISTINCT `created`, `user_id`, `action`, MAX(`id`) AS `id`
+  FROM `log_activity`
+  WHERE DATE(`created`) BETWEEN @WEEK_BEGIN AND @WEEK_END
+  /* The type of review, see constants/activity.py */
+  AND `action` IN (21, 43, 144, 165)
```

Comment:
I know this was present in the original through `note_key`, but I think this should be a parameter instead, so we could use `%s` here and pass `amo.LOG_REVIEWER_REVIEW_ACTION` (which contains more ids but I think that's just because it has obsolete constants as well - we could clean it up) to the `execute()` call and not worry about maintaining this list.

Reply:
added as a parameter https://github.com/mozilla/addons-server/pull/20060/commits/b796def1381f3218806680ff9303ed91a069c9c8 - extra fun because mysql doesn't support `SET` with a list of values so you need to convert the ids to a comma-seperated-list and use a function to check the id is in it.

Can't use `amo.LOG_REVIEWER_REVIEW_ACTION` because it's a mix of content and non-content review actions and these reports split them out, so I just defined the appropriate actions instead.